xla/stream_executor/rocm/BUILD

```diff
@@ -442,6 +447,57 @@ cc_library(
     alwayslink = True,
 )
 
+cc_library(
+    name = "hipblaslt_if_static",
+    deps = if_rocm_hipblaslt([
+        "@local_config_rocm//rocm:hipblaslt",
+    ]),
+)
+
+cc_library(
+    name = "hipblaslt_plugin",
```

Comment:
Please put the hdrs, srcs and deps behind if_rocm_is_configured guards.

Reply:
Done